### PR TITLE
#2868 demo create account button

### PR DESCRIPTION
**Pull request by ackernaut**
#### Demo
- [x] Button routes to `/create-account?from=demo`
- [x] Team invite modal has CTA for demo, button routes to `/create-account?from=demo`
- [x] Layout is adaptive (avatars are centered below header and button for small viewports)
- [x] If a Parabol user visits the demo they don’t see the create account button
- [x] If a Parabol user visits the demo they see a conditional message in team invite modal and a button that takes them to My Dashboard

#### Action/Retro
- [x] Create account button is not there
- [x] Team invite modal works in meeting views
- [x] Layout is adaptive (avatars are pinned top right at all viewport widths)
- [x] Team invite modal works from lobby

#### Team Dash
- [x] Team invite modal works in team dashboard

### CODE REVIEW

**Comment by ackernaut** [comment]
PR ready for review!

File: src/universal/components/AddTeamMemberModal.tsx [hunk 1]
```diff
@@ -122,6 +125,13 @@ class AddTeamMemberModal extends Component<Props, State> {
   render () {
     const {closePortal, error, submitting} = this.props
     const {invitees, successfulInvitations, rawInvitees} = this.state
+    if (isDemoRoute()) {
```
Comment (mattkrick) [inline src/universal/components/AddTeamMemberModal.tsx]: the demo modal doesn't require any of the state or props that this component uses, so it should probably be promoted.
I'd make it a lazy component on AddTeamMemberAvatarButton, that way we only load it for the demo

File: src/universal/components/MeetingContentHeader.tsx [hunk 2]
```diff
@@ -12,26 +15,61 @@ const MeetingContentHeaderStyles = styled('div')({
...
+    flex: 1
+  },
+  [meetingTopBarMediaQuery]: {
+    alignItems: 'flex-start'
+  }
 })
 
+const PrimaryActionBlock = styled('div')({
+  order: 2,
+  [localHeaderBreakpoint]: {
```
Comment (mattkrick) [inline src/universal/components/MeetingContentHeader.tsx]: why create a new row? the hamburger is less than the width of 1 avatar, so we're not saving any space
Screenshot from 2019-06-11 07-14-14

if it is the demo, we know only 3 avatars will be present, so we'll have enough space to fit them.

File: src/universal/components/CreateAccountButton.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+const StyledButton = styled(SecondaryButton)({
+  paddingLeft: 16,
+  paddingRight: 16
+})
+
+const CreateAccountButton = (props: RouteComponentProps) => {
+  const {history} = props
+  const handleClick = () => history.push('/create-account?from=demo')
+  return <StyledButton onClick={handleClick}>{'Create Free Account'}</StyledButton>
```
Comment (mattkrick) [inline src/universal/components/CreateAccountButton.tsx]: what if the viewer is logged in? Either this should say "Go to dashboard" or it shouldn't exist at all

File: src/universal/components/AddTeamMemberAvatarButton.tsx [hunk 4]
```diff
@@ -5,58 +5,33 @@ import styled from 'react-emotion'
...
-        height: 64,
-        width: 64,
-        maxWidth: 64
-      }
-    }
-)
+const AddButton = styled(OutlinedButton)({
+  fontSize: 24,
+  fontWeight: 400,
+  height: 32,
```
[the remark this answers is not in the document]
Comment (ackernaut) [reply src/universal/components/AddTeamMemberAvatarButton.tsx]: I left some comments in Figma about a potential target IA. It takes the full list of avatars out of the control bar, basically. I’m okay letting these grow smaller and larger in the meantime. I’ll look at adding this back in.

File: src/universal/components/AddTeamMemberModal.tsx (hunk 1 above)
Comment (ackernaut) [reply src/universal/components/AddTeamMemberModal.tsx]: Thanks for the tip, I’ll look at this.

File: src/universal/components/MeetingContentHeader.tsx (hunk 2 above)
Comment (ackernaut) [reply src/universal/components/MeetingContentHeader.tsx]: I left comments in Figma, basically it shouldn’t wrap like if there is room. I’ll double check.

File: src/universal/components/CreateAccountButton.tsx (hunk 3 above)
Comment (ackernaut) [reply src/universal/components/CreateAccountButton.tsx]: Agreed, I’ll look for the a to specify that logic.

Comment (ackernaut) [reply src/universal/components/CreateAccountButton.tsx]: @mattkrick not immediately sure on how to do that, so if you see this before I figure it out, I’ll probably need a hint

Comment (mattkrick) [reply src/universal/components/CreateAccountButton.tsx]: i'd just check local storage to see if there's a token present. if so, they're logged in

**Pull request merged**